zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json

```diff
@@ -940,6 +940,11 @@
     "version": {
       "description": "Record version tag.",
       "type": "string"
+    },
+    "custom": {
+      "description": "Custom metadata",
+      "type": "object",
+      "additionalProperties": { "type": "object" }
```

Comment:
I think you can expand this one (based on the spec `additionalProperties` can be either a boolean or a valid JSONSchema), and validate the children as well (i.e. define `value` and `uri` as properties with `required: ['value']`).